test/spotify_test.dart

```diff
@@ -109,7 +125,7 @@ Future main() async {
 
   group('Show episodes', () {
     test('list', () async {
-      var episodes = await spotify.shows.episodes('4AlxqGkkrqe0mfIx3Mi7Xt');
+      var episodes = spotify.shows.episodes('4AlxqGkkrqe0mfIx3Mi7Xt');
```

Comment:
Why remove the `await`? 🤔

Reply:
I removed, because vscode showed me a warning, that the `await` is applied to a `Pages<>` which is not a  `Future`.

Reply:
Makes sense 😄